### PR TITLE
Some styles changed to reduce the space multiple toasts take on screen

### PR DESCRIPTION
**Pull request by Swaathik**

### CODE REVIEW

File: src/components/Menu/SideMenu/index.js [hunk 1]
```diff
@@ -72,6 +72,7 @@ export class SideMenu extends PureComponent /*:: <Props, State> */ {
         <div role="menu">
           {this.props.visible && this.props.showConnectionStatusToast ? (
             <Tip
+              title="💡 How strong are the connections?"
```
Comment (CodeGit) [inline src/components/Menu/SideMenu/index.js:75]: What is the status of external dependencies?

File: src/components/Menu/SideMenu/index.js [hunk 2]
```diff
@@ -72,6 +72,7 @@ export class SideMenu extends PureComponent /*:: <Props, State> */ {
...
               body="The green and red signals of the connection status shows the connectivity strength of the URLs"
```
Comment (CodeGit) [inline src/components/Menu/SideMenu/index.js:76]: The green signal is displayed when an external dependency is available whist a red signal indicates that InterPro cannot connect to the resource.

File: src/pages/Home/index.js [hunk 3]
```diff
@@ -213,13 +213,15 @@ class Home extends PureComponent {
       <>
         {this.props.showSettingsToast ? (
           <Tip
+            title="💡 Where to find the website settings?"
```
Comment (CodeGit) [inline src/pages/Home/index.js:216]: How to edit the website settings?

**Review by CodeGit — APPROVED** [review]

**Comment by Swaathik** [comment]
Cool. I will change the texts

**Pull request merged**